SU-HSDO/suhumsci
# SHS-5625: Implementation: New Accordion Style (BED)

# READY FOR REVIEW

## Summary
Add "Light" option to Accordion component's "Style" field. 

## Steps to Test
1. Add/edit a flexible page
2. Add an Accordion component
3. In the Style select field verify there is a new option for `Light`
4. Verify the class `hb-accordion-light` is applied to the wrapper 
![image](https://github.com/SU-HSDO/suhumsci/assets/25091784/3d92940a-03f9-49e8-bc5b-3f2df859e40c)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
